docs/admin/federation/kubefed.md

```diff
@@ -0,0 +1,172 @@
+---
+assignees:
+- madhusudancs
+
+---
+Kubernetes v1.5 introduced a new command line tool called `kubefed`
+to facilitate the administration of cluster federations. `kubefed`
+stands for "Kubernetes Federate". It aids in both deploying a
+Kubernetes Cluster Federation control plane and adding/removing
+clusters from it. 
+
+This guide explains how to administer a Kubernetes Cluster Federation
+using `kubefed`.
+
+`kubefed` is considered alpha in Kubernetes v1.5.
```

Comment:
Set this off with a blockquote, like below:

"> Note: `kubefed` is an alpha feature in Kubernetes 1.5."